basic_samples/Authentication/AuthorizationCodeFlow/Python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.23.0
+selenium==3.141.0
```

Comment:
Selenium would be only for the testing right? If this is the case I would remove it from the reqs and add it along with pytest if someone is interested in doing that.